lute/static/js/lute.js

```diff
@@ -572,7 +584,14 @@ function _single_tap(el, e) {
   clear_newmultiterm_elements();
   const term_is_status_0 = (el.data("status-class") == "status0");
   if (term_is_status_0) {
-    show_term_edit_form(el);
+    if(_is_mobile_focus){
+      el.addClass('kwordmarked')
+      el.addClass('wordhover')
+      update_status_for_marked_elements(1)
+    }  
+    else{
+      show_term_edit_form(el);
+    }    
```

Comment:
Lots of nested ifs get hard to follow, could be written as:

```
  if (el.data("status-class") != "status0")
    return;
  if (_is_mobile_focus()) { ... etc }
  else { ... }
```

Acknowledging that this might be a personal preference :-)